src/components/MetadataPanel/style.css

```diff
@@ -0,0 +1,84 @@
+.container {
+    color: var(--dark-theme-text-color-bright);
+        padding: 16px 14px 16px 24px;
+}
+
+.collapsed {
+    padding: 14px 8px 8px 16px;
+}
+
+.agent-selected {
+    min-height: 265px;
+}
+
+.title {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 4px;
+}
+
+h3 {
+    color: var(--pure-white);
+    font-weight: 800;
+    margin: 0px;
+}
+
+.icon {
+    background-color: transparent;
+    border: none;
+    color: var(--warm-gray);
+    font-size: 18px;
+}
+
+.title:hover .icon {
+    color: var(--pure-white);
+}
+
+.rotate span {
+    transform: rotate(180deg);
+}
+
+.panel-contents {
+    color: var(--medium-gray);
+}
+
+.no-agent-row {
+    display: flex;
+    flex-direction: column;
+    color: var(--white-two);
+}
+
+.no-agent-row p {
+    margin-bottom: 4px;
+}
+
+.no-agent-text {
+    font-size: 12px;
+    font-weight: 100px;
+}
+
+.agent-missing-text {
+    font-size: 13px;
+    font-weight: 400;
+}
+
+.row {
+    display: flex;
+    font-size: 12px;
+    font-weight: 100;
+}
+
+.key {
+    color: var(--medium-gray);
```

Comment:
there are a few repeated `font-weights` `font-size` and colors (ie you also set the `.panel` color to medium-gray. I would consolidate the styles: figure out what the default font with, size and color are, apply those in general, and then use classes to do overrides when needed. also what's the general app font weight?